ts/store/transforms/dateISO8601Tranform.ts

```diff
@@ -1,6 +1,23 @@
 import { createTransform, TransformIn, TransformOut } from "redux-persist";
 import { DateFromISOString } from "../../utils/dates";
 
+/**
+ * dateFieldsTransformable contains the name of the fields that are
+ * instance of Date and those ones we want to persist/rehydrate in redux persist store
+ *
+ * actually entities state (whitelisted in this transform) contains these following date (Timestamp) fields
+ *
+ * EntitiesState
```

Comment:
Aren't you missing "due_date"?

Reply:
@fpersico Yes, I totally skipped it
added in https://github.com/teamdigitale/io-app/pull/1236/commits/16d8d42696e035eb7af7ba971df6b82b97926275